src/components/CreateNewProjectWizard/Gatsby/SelectStarterList.js

```diff
@@ -131,6 +136,13 @@ const Description = styled.p`
   text-align: justify;
 `;
 
+const SelectionInfo = styled.div`
+  background: ${COLORS.gray[200]};
+  border-radius: 5px;
+  padding: 5px;
+  display: ${props => (props.visible ? 'auto' : 'none')};

```

Comment:
`auto` isn't valid for display :-) We can conditionally set the whole property since `display: block` is the default:

```js
${({ visible }) => !visible && `
  display: none;
`}
```

Reply:
ups, sorry, you're right. 👍 I added the code as you proposed